pkg/proxy/proxy.go

```diff
@@ -411,19 +411,22 @@ func (p *Proxy) ReinstallRoutingRules() error {
 		}
 
 		if !option.Config.EnableIPSec || option.Config.TunnelingEnabled() {
-			if err := removeFromProxyRoutesIPv4(); err != nil {
+			if err := removeFromIngressProxyRoutesIPv4(); err != nil {
 				return err
 			}
 		} else {
 			if err := installFromProxyRoutesIPv4(node.GetInternalIPv4Router(), defaults.HostDevice); err != nil {
 				return err
 			}
 		}
+		if err := removeFromEgressProxyRoutesIPv4(); err != nil {
+			return err
+		}
```

Comment:
This feels a bit awkward, as `removeFromProxyRoutesIPv4()` already deletes the whole `RouteTableFromProxy` table. 

Should we maybe put the call to `removeFromEgressProxyRoutesIPv4()` *prior* to the `if !option.Config.EnableIPSec || option.Config.TunnelingEnabled()` handling?

Reply:
Sure!